test/spec/modules/improvedigitalBidAdapter_spec.js

```diff
@@ -566,6 +566,70 @@ describe('Improve Digital Adapter Tests', function () {
       expect(payload.imp[0].ext.gpid).to.not.exist;
       expect(payload.imp[0].instl).to.not.exist;
     });
+
+    it('should not set site when app is defined in FPD', function () {
+      const getConfigStub = sinon.stub(config, 'getConfig');
+      getConfigStub.withArgs('ortb2.app').returns({ content: 'XYZ' });
+      let request = spec.buildRequests([simpleBidRequest], bidderRequest)[0];
+      let payload = JSON.parse(request.data);
+      expect(payload.site).does.not.exist;
+      expect(payload.app).does.exist;
+      expect(payload.app.content).does.exist.and.equal('XYZ');
+      getConfigStub.restore();
+    });
+
+    it('should not set site when app is defined in CONFIG', function () {
+      const getConfigStub = sinon.stub(config, 'getConfig');
+      getConfigStub.withArgs('app').returns({ content: 'XYZ' });
+      let request = spec.buildRequests([simpleBidRequest], bidderRequest)[0];
+      let payload = JSON.parse(request.data);
+      expect(payload.site).does.not.exist;
+      expect(payload.app).does.exist;
+      expect(payload.app.content).does.exist.and.equal('XYZ');
+      getConfigStub.restore();
+    });
+
+    it('validate site object', function () {
```

Comment:
* following the established convention, the test name begins with "should", for exmple it('should set correct site params')
* please add test for getConfig('pageUrl')